packages/notarization/manager.go

```diff
@@ -153,6 +145,16 @@ func (m *Manager) LoadSnapshot(snapshot *ledger.Snapshot) {
 		panic("could not set last confirmed epoch index")
 	}
 
+	// We set it to the last committed ei. It will be updated upon first confirmed message will arrive.
+	if err := m.epochCommitmentFactory.storage.SetLatestCommittableEpochIndex(snapshot.DiffEpochIndex); err != nil {
+		panic("could not set latest committable epoch index")
+	}
+
+	// We set it to the next epoch after snapshotted one. It will be updated upon first confirmed message will arrive.
+	if err := m.epochCommitmentFactory.storage.SetCurrentEpochIndex(snapshot.DiffEpochIndex + 1); err != nil {
```

Comment:
Why do we have to store this? We can always obtain the current Epoch index based on our clock. There is a `CurrentIndex` method in `epoch/types.go`

Reply:
To know that we moved to the next epoch, we need to see that index has increased